csrc/transformer/dropout_kernels.cu

```diff
@@ -1,5 +1,7 @@
 #include "custom_cuda_layers.h"
 
+const int unroll_factor = 4;

```

Comment:
assert that unroll_factor is 4 since the code is hardcoded with 4 way unrolling.

Reply:
great point, I will do that